addons/account_edi_ubl/models/account_edi_format.py

```diff
@@ -121,12 +122,17 @@ def _get_ubl_namespaces():
                 attachment_data = element.xpath('cac:Attachment//cbc:EmbeddedDocumentBinaryObject', namespaces=namespaces)
                 if attachment_name and attachment_data:
                     text = attachment_data[0].text
+                    name = attachment_name[0].text
+                    # Because some e-fff emitters put the path of the xml instead of the pdf's name,
+                    # we get only the filename and change the extension because we are nice people.
+                    name = PureWindowsPath(name).stem + '.pdf'
```

Comment:
Maybe inline `name` in here?

Also fluff up the comment with the sort of paths you see in order to clarify the why? Or expound on it in the commit message and remove the comment. I know not everyone agrees but I like commit messages to explain the observations & background of a change in details when the change is not self-explanatory.

Reply:
I personally like comments in the code if it's not self-explanatory which is the case here since we only fix the mess made by other accounting softwares. But I can also detail the commit message a bit more.

Reply:
Didn't inline `name` in the end because we need to unescape the name so it would be too much operations in one line.

Reply:
> I personally like comments in the code if it's not self-explanatory which is the case here since we only fix the mess made by other accounting softwares. But I can also detail the commit message a bit more.

I don't mind having it in comments, but please explain more clearly why it's needed e.g. the sort of paths you see from third-party systems (are they always windows-style? sometimes? others?)

Reply:
> Didn't inline `name` in the end because we need to unescape the name so it would be too much operations in one line.

That doesn't make sense. Is it double-escaped in the document text? Escaping should be a serialisation concern, once the document is parsed the actual text should be what's retrieved:
```python
>>> from lxml import etree
>>> etree.fromstring("<foo>bar&amp;baz</foo>").text
'bar&baz'
```

Reply:
Oh yes, this is is stupid --"

Reply:
> > I personally like comments in the code if it's not self-explanatory which is the case here since we only fix the mess made by other accounting softwares. But I can also detail the commit message a bit more.
> 
> I don't mind having it in comments, but please explain more clearly why it's needed e.g. the sort of paths you see from third-party systems (are they always windows-style? sometimes? others?)

I changed the comment and the commit message, is it clear enough now ?

Reply:
Sure.